optimize/pom.xml

```diff
@@ -691,6 +690,30 @@
       </snapshots>
       <layout>default</layout>
     </repository>
+
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>camunda-cpm</id>
```

Comment:
should this be `camunda-bpm`?

Reply:
I thought it would be that too, but `camunda-cpm` is used for this repository in the monorepo, `camunda-bpm` is not used